src/client/screens/OrganizationScreen.tsx

```diff
@@ -223,26 +223,49 @@ const OrganizationScreen = ({ organization, user }: StateProps) => {
                 ) : (
                   <Tooltip
                     key={1}
-                    content={<div>You must confirm your email before joining an organization</div>}
+                    content={
+                      <div>
+                        {userInOrg
+                          ? "Confirm your email to finish joining this organization"
+                          : "You must confirm your email before joining an organization"}
+                      </div>
+                    }
                   >
                     {joinButton}
                   </Tooltip>
                 )
               ) : (
-                joinButton
+                <div></div>
```

Comment:
I don't think you meant to get rid of the Join button for logged out users?
```suggestion
                joinButton
```